.gitignore

```diff
@@ -6,5 +6,5 @@ project/project
 *.*~
 
 # allow
-!/my-talk/target/scala-2.12/mytalk-fastopt.js
-!/my-talk/target/scala-2.12/mytalk-jsdeps.js
\ No newline at end of file
+!/my-talk/target/scala-2.13/mytalk-fastopt.js
+!/my-talk/target/scala-2.13/mytalk-jsdeps.js
```

Comment:
Unfortunately, this project doesn't specify a Scala version (yet). That means, me running on 2.12 won't be able to commit talks to his/her fork anymore. Well, the same is true right now with people on 2.13. Maybe fixing the Scala version solves this issue.